src/Base/ConsoleObserver.h

```diff
@@ -57,15 +57,16 @@ class BaseExport ConsoleObserverStd: public ILogger
 public:
     ConsoleObserverStd();
     ~ConsoleObserverStd() override;
-    void SendLog(const std::string& message, LogStyle level) override;
+    void SendLog(const std::string& notifiername, const std::string& message, LogStyle level) override;
     const char* Name() override {return "Console";}
 protected:
     bool useColorStderr;
 private:
-    void Warning(const char *sWarn);
-    void Message(const char *sMsg);
-    void Error  (const char *sErr);
-    void Log    (const char *sErr);
+    void Warning        (const char *sWarn);
+    void Message        (const char *sMsg);
+    void Error          (const char *sErr);
+    void Log            (const char *sLog);
+    void CriticalMessage(const char *sCriticalMsg);

```

Comment:
What about calling this just Critical()? For error or log messages we also only call them Error() or Log()

Reply:
It is ok. I can bring this change when rebasing. Let me know when you think it is ready.

Reply:
> Let me know when you think it is ready.

From my point of view the PR has no serious issues. However, before merging it we should make sure that it also compiles on Windows (MSVC and MSYS2).